patches/chrome-browser-BUILD.gn.patch

```diff
@@ -10,11 +10,11 @@ index 98ae3acaf7cf5fbfae0e4c2e0ddb8a087e6d6e23..fd339ae93d51f85cb850035d6bc0fc89
    ]
  
    if (build_with_tflite_lib) {
-@@ -3878,6 +3879,7 @@ static_library("browser") {
-         "task_manager/sampling/shared_sampler_posix.cc",
+@@ -3879,6 +3880,7 @@ static_library("browser") {
        ]
      }
-+    sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps
```

Comment:
I placed this in if (!is_android) { } because brave_chrome_browser_importer_sources doesn't support Android. But, brave_chrome_browser_sources|dep should be used by other sources.gni files which should also support Android. I move this to out of if (!is_android) { }, then add the guard to //brave/browser/sources.gni.